src/mixins/document-mixin.ts

```diff
@@ -0,0 +1,94 @@
+import Vue from 'vue'
```

Comment:
This was copied from Create UI and then the axios calls were moved to Legal Services.